### PR TITLE
Increase overall coverage to 92 percent

### PR DESCRIPTION
**Pull request by Wolfy7**
@burntcarrot 
I add some new test cases to increase the coverage to 92%

grafik

### CODE REVIEW

File: absorb/main.py [hunk 1]
```diff
@@ -12,7 +12,6 @@
 @click.group()
 def cli() -> None:
     """Creates the main click group for absorb."""
```
Comment (burntcarrot) [inline absorb/main.py:14]: Can you please let me know that is this working while running locally?

I'm a bit skeptical on the removal of `pass` (although it should work fine)

**Review by burntcarrot — COMMENTED** [review]
Looks good to me! Just some minor changes, can you look at the comments below.

File: absorb/main.py (hunk 1 above)
Comment (Wolfy7) [reply absorb/main.py:14]: Yes, this works. Tested locally.
If you dont want to removed it, the line will be shwon as Missing in the coverage.

Comment (burntcarrot) [reply absorb/main.py:14]: Thanks for the heads up!

**Comment by burntcarrot** [comment]
Looks good to me!

Thanks for your contribution @Wolfy7! Welcome to Hacktoberfest 2021! ❤

**Comment by Wolfy7** [comment]
@burntcarrot  Thank you for merging and also for giving the possibility to contribute.
it was a pleasure for me :) ❤

**Pull request merged**